Sources/AccessibilitySnapshot/Core/Swift/Classes/UIAccessibility+EffectiveProperties.swift

```diff
@@ -0,0 +1,227 @@
+//
+//  Copyright 2024 Block Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+// TODO: Do the block variants automatically account for the attributed versions into the non-attributed versions?
+
+extension NSObject {
+
+    var effectiveIsAccessibilityElement: Bool {
+        #if swift(>=5.9)
+        if #available(iOS 17.0, *) {
+            return isAccessibilityElementBlock?() ?? isAccessibilityElement
+        } else {
+            return isAccessibilityElement
+        }
```

Comment:
you should remove 'else' block, you have 'return isAccessibilityElement' in the end, this is duplicate. 
And in other variables same